src/pybind/mgr/dashboard/README.rst

```diff
@@ -77,7 +77,7 @@ dashboard, you will need to provide credentials. If you do not have a user
 which shall be used for providing those credentials, you will also need to
 create one::
 
-  $ radosgw-admin user create --uuid=<user> --display-name=<display-name> \
```

Comment:
Consider removing that patch, as I already corrected that typo and moved this paragraph from the README to the reference manual in #21443

Reply:
Done